zuul.d/jobs.yaml

```diff
@@ -25,3 +25,21 @@
       - ^docs_dev/.*$
       - ^docs_user/.*$
       - Gemfile
+
+- job:
+    name: data-plane-adoption-osp-17-to-extracted-crc-ironic
+    parent: data-plane-adoption-osp-17-to-extracted-crc
+    vars:
+      use_ceph: "false"
+      dpa_test_suite: "test-with-ironic"
+    irrelevant-files:
+      - ^.*\.md$
+      - ^.*\.rst$
+      - ^docs/.*$
+      - .github/workflows
+      - mkdocs.yml
+      - .gitignore
+      - LICENSE
+      - ^docs_dev/.*$
+      - ^docs_user/.*$
+      - Gemfile
```

Comment:
adding note for discussion with @cescgina 
I think maybe we want to move these definitions into rdo-jobs with the other definitions? For example, if we want to add these jobs to the other repos (ci-framework/install_yamls etc) might be better for them to live in rdo-jobs.

we can merge this and do it after, just adding the note so we can discuss at some point

Reply:
It might be a good idea since it'll make the inheritance tree simpler. Zuul will not care, but someone approaching from zero could easily get confused because we have parenting relationships between repos in multiple directions